tour/getting-started.md

```diff
@@ -2,4 +2,29 @@
 title: Getting Started
 ---
 
-Talk about how to get started.
+First make sure you have Java, Maven and Git installed on your machine.  Oh you are already rocking? OK let's go!
+
+1. Clone the tour onto your machine:
+```commandline
+git clone -b tour https://github.com/apache/accumulo-website.git tour
+cd tour
+```
+2. Open Main.java in your favorite editor.
+```commandline
+vim ./src/main/java/tour/Main.java
+```
+Notice the main method creates a MiniAccumuloCluster with a root password of "tourguide".  MiniAccumuloCluster is a mini
```

Comment:
Could mention  that it stores files in `target/` in temp that starts with `mac`.  Could also add comment about this in tour code.

Reply:
Yeah good call.  I put this in the README but wouldn't hurt to mention on the tour as well.